clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/ops/servergroup/DisableDcosServerGroupAtomicOperation.groovy

```diff
@@ -34,7 +35,7 @@ class DisableDcosServerGroupAtomicOperation implements AtomicOperation<Void> {
     def resizeDesc = new ResizeDcosServerGroupDescription().with {
       region = description.region
       serverGroupName = description.serverGroupName
-      instances = 0
+      capacity = new ServerGroup.Capacity(desired: 0)
```

Comment:
I know we're doing this because certain pipeline stages in orca call resize the resize operation and assume it takes a capacity, but this has the side effect of exposing capacity for our normal resize operation as well, when really we don't support the min and max fields on that object, just desired. 

What are your thoughts about handling this similar to the cloud foundry implementation, which handles the pipeline stages capacity if it is supplied, but maintains the resize description object as just what is supported by the operation itself? (basically massaging the capacity desired value into the targetSize field)